src/policy/policy.cpp

```diff
@@ -142,8 +142,12 @@ bool IsStandardTx(const CTransaction& tx, const std::optional<unsigned>& max_dat
             reason = "bare-multisig";
             return false;
         } else if (IsDust(txout, dust_relay_fee)) {
-            reason = "dust";
-            return false;
+            // We ensure ephemeral output is spent in the same mempool package
```

Comment:
suggested in other PR: "switch the ordering? else if (whichType != ANCHOR && IsDust())"